### PR TITLE
DetailsList - style updates

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ yarn change`

#### Description of changes

Azure framework requested styling updates DetailsList grid to match toolkit.

### CODE REVIEW

File: packages/azure-themes/src/azure/styles/DetailsList.styles.ts [hunk 1]
```diff
@@ -53,16 +63,16 @@ export const DetailsListStyles = (props: IDetailsListStyleProps): Partial<IDetai
...
           borderTop: StyleConstants.borderNone,
           selectors: {
             '.ms-DetailsHeader-cellTitle': {
               backgroundColor: semanticColors.listBackground,
               borderRightWidth: StyleConstants.borderWidth,
               borderRightStyle: StyleConstants.borderSolid,
               borderRightColor: semanticColors.listBackground,
-              fontSize: FontSizes.size10,
-              fontWeight: 700,
+              fontSize: FontSizes.size13,
```
Comment (q-xg) [inline packages/azure-themes/src/azure/styles/DetailsList.styles.ts:74]: I don't think it in use as it is always overwritten by cellName style.

**Comment by JustSlone** [comment]
Close/Reopen to re-run the build and checks

**Comment by JustSlone** [comment]
@Jacqueline-ms @q-xg just checking in on this review. Is there anything you need from us to help move this along?

**Comment by Jacqueline-ms** [comment]
I created a new PR with these updates + more as I had upstream issues with this, I am going to close this PR as it is outdated. 

You can find updates here instead - https://github.com/microsoft/fluentui/pull/14014

**Pull request closed without merging**